terraspace/app/stacks/peer/tfvars/us-west-2/prod.tfvars

```diff
@@ -0,0 +1,6 @@
+vpc = {
+  name = "test-ipfs-peer-subsys"
+}
+cluster_version          = 1.21
```

Comment:
this is already set in the base

Reply:
Removed

Reply:
Cluster version is now part of the EKS object. 
Terraspace layer thing unfortunately doesn't work for "mixing" attributes of the same objects that have different values on each layer. The whole object needs to be re-specified. Even though we loose that possibility when grouping stuff I think it is worth it because keep the object consistent with the actual resource being created.